src/app/forms/templates/test-records/section-templates/test/desk-based-test-section-group2-PSV.ts

```diff
@@ -0,0 +1,146 @@
+import { ValidatorNames } from '@forms/models/validators.enum';
+import { FormNode, FormNodeEditTypes, FormNodeTypes, FormNodeViewTypes, FormNodeWidth } from '@forms/services/dynamic-form.types';
+
+export const DeskBasedTestSectionGroup2PSV: FormNode = {
+  name: 'testSection',
+  label: 'Test',
+  type: FormNodeTypes.GROUP,
+  children: [
+    {
+      name: 'testStartTimestamp',
+      label: 'Test start date',
+      type: FormNodeTypes.CONTROL,
+      value: new Date().toISOString(),
```

Comment:
Nice idea but we try not to put functions into our templates to keep them strictly json. Instead, we set these dates in the effect or resolver that handle creating a test.